### PR TITLE
[viewer] add metrics to grafana

### PR DESCRIPTION
**Pull request by phoenix-xhuang**
usage:

`docker-compose -f docker-compose.yml  -f docker-compose.dev.yml down --remove-orphans  && docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d`

default:
http://ip:13000
admin/admin

### CODE REVIEW

**Review by fenrir-z — DISMISSED** [review]

**Review by IJtLJZ8Rm4Yr — DISMISSED** [review]

**Comment by phoenix-xhuang** [comment]
screenshot
91391660972791_ pic_hd

**Comment by phoenix-xhuang** [comment]
screenshot
75001660980634_ pic_hd

File: metrics/config/loki.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  storage:
+    type: local
+    local:
+      directory: /etc/loki/rules
+  rule_path: /etc/loki/rules-temp
+  alertmanager_url: http://localhost:9093
+  ring:
+    kvstore:
+      store: inmemory
+  enable_api: true
```
Comment (fenrir-z) [inline metrics/config/loki.yml]: 空行

File: metrics/config/promtail.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - url: http://loki:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: system
+  static_configs:
+  - targets:
+      - localhost
+    labels:
+      job: varlogs
+      __path__: /var/log/*log
```
Comment (fenrir-z) [inline metrics/config/promtail.yml]: 空行

File: metrics/config/loki.yml (hunk 1 above)
Comment (phoenix-xhuang) [reply metrics/config/loki.yml]: updated

File: metrics/config/promtail.yml (hunk 2 above)
Comment (phoenix-xhuang) [reply metrics/config/promtail.yml]: updated

**Review by fenrir-z — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**